### ISSUE TITLE
Comparison to porteurbars

### ISSUE THREAD

**Issue by kskyten** [ffizer/ffizer#80]
The two projects seem quite similar. It would be useful to have a comparison. Here is a short list:

- I'm not sure whether porteurbars templates can import other templates
- I prefer treating templates as local files by default instead of having to type file://.... Ideally this would be handled in a smart way so that a github lookup would be a fallback for non-existent files, for example.
- Very subjective, but I prefer the name ffizer.
- ffizer has a handy gitignore.io helper function
- ffizer has more recent development activity
- Overall, I prefer the UI of porteurbars except for the url handling (see my other issues).

Any other differences?

**Comment by davidB** [comment]
Good catch, I didn't see the value of a detailed comparison (in form table/list) with alternatives. But PR is welcome. TBH I think I should clarify/simplify the features list.
I discovered porteurbars few days after the start (but too late, and also planned some features out of scope of porteurbars).
FYI, like the helper function for gitignore.io, an other is planned to interact with spdx to allow creation of LICENSE file.

The ability to compose/import/layer with other templates is a "unique" feature (I didn't find other system with that).

Github lookup as fallback is a good idea, need to thinks about it (I used ffizer with bitbucket and gitlab repo also).

**Comment by kskyten** [comment]
Another possibility for the fallback is to have a registry repo where all the templates are registered (see https://github.com/JuliaRegistries/General for an example). This way the templates can be hosted anywhere. A centralized location would also improve the discoverability of the templates.

**Comment by davidB** [comment]
The central and local registry for alias is planned. 
I should write the roadmap or the list of planned features :-P
